hangman.rb

```diff
@@ -0,0 +1,124 @@
+class Hangman
+
+  def initialize
+    @now_playing = true
+    @count = 0
+
+    # Board Display
+    @row1 = "|     _________"
+    @row2 = "|     |/      |"
+    @row3 = "|     |      "
+    @row4 = "|     |      "
+    @row5 = "|     |       "
+    @row6 = "|     |      "
+    @row7 = "|     |"
+    @row8 = "| ____|_____"
+
+    @board = [@row1, @row2, @row3, @row4, @row5, @row6, @row7, @row8]
+
+    @dictionary = ["hotdog", "elephant", "computer", "developer", "baby", "dragon", "classmate", "program", "teamwork", "fun"]
+
+    start
+  end
+
+  def start
+    puts "Let's play a game of Hangman!"
+    puts @board
+
+    pick_word
+    @word = ["_ "] * @answer_letters.length
+    puts @word.join
+    play
+  end
+
+  def play
+    while @now_playing == true
+      get_guess
+      check_guess
+      puts @board
+      puts @word.join
+      win
+    end
+  end
+
+  def pick_word
+    #randomly selects word
+    answer = @dictionary[rand(0...@dictionary.length)]
+    #convert individual letters of word to array
```

Comment:
@roseaboveit thank you! This feels like a lifetime ago, but I appreciate the feedback. I thought I was controlling for that error by doing a range (0...@dictionary.length) with the three dots as opposed to the two. I am not sure how I'd remedy this at this point. Mostly because I think I'd approach the entire assignment differently and so it's hard to return to that frame of mind.